auth/src/main/java/com/firebase/ui/auth/data/remote/TwitterSignInHandler.java

```diff
@@ -50,6 +45,18 @@ public TwitterSignInHandler(Application application) {
         mClient = new TwitterAuthClient();
     }
 
+    public static void initializeTwitter() {
+        Context context = AuthUI.getApplicationContext();
+
+        // Note: this has no effect if Twitter is already initialized so it's safe to call
+        // multiple times.
```

Comment:
I'm wondering if it makes sense to do what we had for the `PhoneNumberUtils`: move the initialization back to the static block and have the initialize method be empty with a comment saying it's forcing static init. What do you think? It just feel weird to be constantly initialising Twitter even though I know it doesn't do anything.

Reply:
If we did that, how would we make initialization happen before `signOut()` is called without getting a ton of bad-class log spam in the case where Twitter deps are not included?

Reply:
SGTM